lib/src/widget/menu/app_menu.dart

```diff
@@ -0,0 +1,152 @@
+import 'package:flutter/material.dart';
+import 'package:xayn_design/src/linden/linden.dart';
+import 'package:xayn_design/src/widget/unter_den_linden/unter_den_linden.dart';
+
+const _kDividerThickness = 1.0;
+
+class AppMenu extends StatelessWidget {
```

Comment:
Would be good to add a comment what is `AppMenu` and what it does.

Reply:
[a060c10](https://github.com/xaynetwork/xayn_design/pull/69/commits/a060c10fbff75619a61ccb7ac0e144e1461acc70)